openc3-cosmos-init/plugins/packages/openc3-tool-base/public/index-allow-http.html

```diff
@@ -0,0 +1,100 @@
+      Learn more about CSP policies at https://content-security-policy.com/#directive
+    -->
+  <meta http-equiv="Content-Security-Policy"
+    content="default-src 'self' blob: data: http: https:; script-src 'unsafe-inline' 'unsafe-eval' http: https: blob:; connect-src blob: http: https: wss: ws:; style-src 'unsafe-inline' http: https:; object-src 'none';" />
+
+  <meta name="importmap-type" content="systemjs-importmap" />
+  <!-- If you wish to turn off import-map-overrides for specific environments (prod), uncomment the line below -->
+  <!-- More info at https://github.com/joeldenning/import-map-overrides/blob/master/docs/configuration.md#domain-list -->
+  <!-- <meta name="import-map-overrides-domains" content="denylist:prod.example.com" /> -->
+
+  <!-- Shared dependencies go into this import map. Your shared dependencies must be of one of the following formats:
+
+      1. System.register (preferred when possible) - https://github.com/systemjs/systemjs/blob/master/docs/system-register.md
+      2. UMD - https://github.com/umdjs/umd
+      3. Global variable
+
+      More information about shared dependencies can be found at https://single-spa.js.org/docs/recommended-setup#sharing-with-import-maps.
+
+      Update by downloading latest release from https://github.com/vuetifyjs/vuetify/releases (for example)
+      and putting the vuetify-vX.Y.Z.min.js in ../public/js (delete the old one)
+    -->
+  <script type="systemjs-importmap">
+    {
+      "imports": {
+        "single-spa": "/js/single-spa-5.9.5.min.js",
+        "vue": "/js/vue.global.prod-3.5.13.js",
+        "vuetify": "/js/vuetify-labs-3.7.4.js",
+        "vue-router": "/js/vue-router-4.4.4.min.js",
+        "vuex": "/js/vuex-4.1.0.min.js"
+      }
+    }
+  </script>
+  <script type="systemjs-importmap" src="/openc3-api/map.json"></script>
+  <!-- See note above for how to update these -->
+  <link rel="preload" href="/js/single-spa-5.9.5.min.js" as="script" />
+  <link rel="preload" href="/js/vue.global.prod-3.5.13.js" as="script" />
+  <link rel="preload" href="/js/vuetify-labs-3.7.4.js" as="script" />
+  <link rel="preload" href="/js/vue-router-4.4.4.min.js" as="script" />
+  <link rel="preload" href="/js/vuex-4.1.0.min.js" as="script" />
+  <script src="/js/auth.js"></script>
+  <script src="/js/import-map-overrides-3.1.1.min.js"></script>
+  <!-- systemjs homepage: https://github.com/systemjs/systemjs/releases
+      We use system.js instead of s.js which includes global and module-types.
+      We also explicitly use amd (asynchronous module definition) which uses named-register
+      Note that named-exports, transform and use-default are deprecated.
+    -->
+  <script src="/js/systemjs-6.15.1.min.js"></script>
+  <!-- All the following belong to the system package -->
+  <script src="/js/amd.min.js"></script>
+  <script src="/js/named-register.min.js"></script>
+</head>
```

Comment:
We'll have to update `package_audit_lib.rb` to look for this new file and update it appropriately